recipe/bld.bat

```diff
@@ -4,8 +4,16 @@ if "%VS_MAJOR%" == "9" (
     set MANIFEST=yes
 )
 
+:: Get libxml2 include paths via pkg-config
+:: N.B.: We may at some point want to try another build system that supports
+::       pkg-config out of the box.
+set "PKG_CONFIG_PATH=%LIBRARY_LIB%\pkgconfig;%LIBRARY_PREFIX%\share\pkgconfig"
```

Comment:
They seem to have a CMake based build process too. It might be worthwhile checking it out later.